.pre-commit-config.yaml.jinja

```diff
@@ -98,6 +98,8 @@ repos:
     rev: v2.6.0
     hooks:
       - id: prettier
+       {% if prettier_exclude %}
+       exclude: {{ prettier_exclude }}
```

Comment:
In Jinja, [`{% if %}` tags](https://jinja.palletsprojects.com/en/3.1.x/templates/#if) need to be closed by the `{% endif %}` tag.